Gems/MachineLearning/Code/Source/Nodes/SupervisedLearning.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <Nodes/SupervisedLearning.h>
+#include <Models/MultilayerPerceptron.h>
+#include <MachineLearning/ILabeledTrainingData.h>
+#include <Algorithms/Training.h>
+#include <AzCore/Console/ILogger.h>
+#include <AzCore/std/chrono/chrono.h>
+
+namespace MachineLearning
+{
+    INeuralNetworkPtr SupervisedLearning::In
+    (
+        INeuralNetworkPtr Model,
+        ILabeledTrainingDataPtr TrainingData, 
+        ILabeledTrainingDataPtr TestData,
+        //LossFunctions CostFunction, 
```

Comment:
Remove commented-out code

Reply:
This one needs incremental feature improvements from script canvas, once it supports enum inputs I can drop the raw integral pin and make it something actually meaningful to people. Right now they have to double check code and count which integer corresponds to the enum value they want, and then hand enter that integer into the pins value.